tlatools/org.lamport.tlatools/src/tlc2/tool/Worker.java

```diff
@@ -96,12 +96,7 @@ public void run() {
 				curState = this.squeue.sDequeue();
 				if (curState == null) {
 					synchronized (this.tlc) {
-						if(!this.tlc.setDone()) {
-							final int ec = tool.checkPostCondition();
-							if (ec != EC.NO_ERROR) {
-								tlc.setError(true, ec);
-							}
-						}
+						this.tlc.setDone();
```

Comment:
This is what this section of Worker.java looked like before the introduction of the POSTCONDITION feature.